src/content/docs/reference/overview.mdx

```diff
@@ -9,6 +9,14 @@ sidebar:
 
 import { LinkCard, CardGrid } from '@astrojs/starlight/components';
 
+To the Iris docs!
+
+Iris is a shader loader that is integrated with sodium, with support for modern shaders, built from the ground up for ease of use. Weather your new to shader development, want to migrate your shader from optifine, or just need a refresher on what Iris can do, you have come to the right place. 
```

Comment:
```suggestion
Iris is a shader loader that is integrated with [Sodium](https://modrinth.com/mod/sodium/), with support for modern shaders, built from the ground up for ease of use. Whether you're new to shader development, want to migrate your shader from Optifine, or just need a refresher on what Iris can do, you can find everything you need to know right here.
```

Reply:
Also I feel like "modern shaders" is vague and doesn't mean much.

Reply:
Sounds good tho